src/main/java/seedu/address/ui/PersonPane.java

```diff
@@ -60,54 +60,85 @@ public PersonPane() {
      */
     private void displayPersonDetails() {
         if (person != null) {
```

Comment:
Maybe you can consider using a guard clause here for the person, but its just a minor detail.